legend-sdlc-test-reports/pom.xml

```diff
@@ -219,16 +219,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-<!--        <dependency>-->
```

Comment:
It's better to leave this in and commented out rather than deleting it. The fact that we're commenting it out is a workaround.